spring-pulsar/src/test/java/org/springframework/pulsar/listener/PulsarListenerTests.java

```diff
@@ -268,8 +268,8 @@ public RedeliveryBackoff redeliveryBackoff() {
 	@ContextConfiguration(classes = DeadLetterPolicyTest.DeadLetterPolicyConfig.class)
 	class DeadLetterPolicyTest {
 
-		static CountDownLatch latch = new CountDownLatch(2);
-		static CountDownLatch dlqLatch = new CountDownLatch(1);
+		private static final CountDownLatch latch = new CountDownLatch(2);
```

Comment:
I brought this up in a [PR recently](https://github.com/spring-projects-experimental/spring-pulsar/pull/85#discussion_r962250470) w/ @sobychacko and in other Spring projects we do not use `final` everywhere. I think we should stay consistent w/ that approach here as well. 

For now, let's not add `final` until we decide what we want to go w/.  Also, whatever we decide we should probably add/configure the format/checkstyle to enforce it.